.github/workflows/e2e.yml

```
@@ -133,6 +133,9 @@ jobs:
         run: python simulation.py
       - name: Run driver test
         run: ./../test_driver.sh "${{ matrix.directory }}"
+      - name: Test server with database
+        if: ${{ matrix.directory == 'bare' }}
+        run: ./../test_driver.sh bare db
```

We will add support for additional databases, should we rename this to `sqlite`?
```suggestion
        run: ./../test_driver.sh bare sqlite
```

---

Yes, correct, I updated it